packages/tests/package.json

```diff
@@ -20,8 +24,11 @@
   "dependencies": {
     "@slashid/slashid": "^3.22.0",
     "@types/uuid": "^9.0.8",
+    "axios": "^1.6.8",
+    "bcryptjs": "^2.4.3",
     "cheerio": "1.0.0-rc.12",
     "dotenv": "^16.4.5",
+    "form-data": "^4.0.0",
```

Comment:
@kaspermroz this looks accidental, can you check please?

Reply:
This is for the autogenerated client, similarly to `axios`